Asgardia.Constitution.Proposal.txt

```diff
@@ -13,27 +13,27 @@ CHAPTER 1.
 DECLARATION OF UNITY OF ASGARDIA
 
 We, the free people of
-the first in the history of existing humanity
-Space State
-ASGARDIA,
+the first Space Nation in the history of existing humanity,
+Asgardia,
 based on the need to expand into the Universe,
 adopt this
-DECLARATION
-§ 1. Asgardia is a free and unified space state. 
+Declaration
+
+§ 1. Asgardia is a free and unified space nation. 
 § 2. The objectives of Asgardia are to:
-a. ensure peace in space;
-b. ensure the protection of humankind from outer space threats;
-c. ensure equal opportunities in space for all Asgardians living on Earth, regardless of geographic, financial, technological and other features of the countries of their earthly citizenship.
-§ 3. Any citizen of Earth can become a citizen of Asgardia, if he/she concurs with this Declaration, respects the Constitution and legislation of Asgardia. 
-§ 4. All citizens of Asgardia are equal, regardless of their country of origin, residence, citizenship, race, nation, gender, language, or financial standing. 
-§ 5. Asgardia respects the laws of the states on Earth and the international agreements and wishes to be recognized as equal country among other states on Earth. 
-§ 6. Asgardia does not interfere in the affairs of the states on Earth on the principle of reciprocity.
+ a. ensure peace in space;
+ b. ensure the protection of humankind from outer space threats;
+ c. ensure equal opportunities in space for all Asgardians living on Earth, regardless of geographic, financial, technological, and other features of the countries of their earthly citizenship.
+§ 3. Any citizen of Earth can become a citizen of Asgardia, if he/she concurs with this Declaration, and respects the Constitution and legislation of Asgardia. 
+§ 4. All citizens of Asgardia are equal, regardless of their place of birth, current residence, language, gender, sexual orientation, race, nationality, creed, financial standing, or citizenship of Earth nations. 
+§ 5. Asgardia respects the laws and international agreements of the nation states on Earth and wishes to be recognized as an equal country among other nation states on Earth. 

```

Comment:
I used "on" from the perspective that we will eventually be running Govt. from Space. If we are strictly on Earth, then "of" would make for better usage.